python/xorbits/_mars/dataframe/groupby/core.py

```diff
@@ -513,7 +513,10 @@ def execute(cls, ctx, op: "DataFrameGroupByOperand"):
                 level=op.level,
                 as_index=op.as_index,
                 sort=op.sort,
-                group_keys=op.group_keys if op.group_keys is not None else no_default,
+                # For GPU, unexpected problems would happen if ``group_keys=no_default``.
```

Comment:
Could you be more specific about the problem if `group_keys`was set to `no_default`?

Reply:
Add detailed error on code comments.